libafl/src/executors/inprocess/mod.rs

```diff
@@ -437,15 +424,14 @@ pub fn run_observers_and_save_state<E, EM, I, OF, S, Z>(
 /// # Safety
 /// This will directly access `GLOBAL_STATE` and related data pointers
 #[cfg(any(unix, feature = "std"))]
-pub unsafe fn generic_inproc_crash_handler<E, EM, I, OF, S, Z>()
+pub unsafe fn generic_inproc_crash_handler<E, EM, I, OF, S>()
 where
-    E: Executor<EM, I, S, Z> + HasObservers,
+    E: Executor<EM, I, OF, S> + HasObservers,
     E::Observers: ObserversTuple<I, S>,
     EM: EventFirer<I, S> + EventRestarter<S>,
     OF: Feedback<EM, I, E::Observers, S>,
     S: HasExecutions + HasSolutions<I> + HasCurrentTestcase<I>,
     I: Input + Clone,
-    Z: HasObjective<Objective = OF> + ExecutionProcessor<EM, I, E::Observers, S>,
```

Comment:
This is the only place where the `ExecutionProcessor` constraint appears. Things seem to work without it, so I'm not sure why it's there.